pkg/apis/openebs.io/scp/v1alpha1/storage-cohort.go

```diff
@@ -33,145 +33,72 @@ type StorageCohort struct {
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 
 	// Spec defines the behavior of a cohort.
+	// +optional
 	Spec StorageCohortSpec `json:"spec,omitempty"`
 
 	// Most recently observed status of the cohort.
 	// Populated by the cohort operator or cohort manager.
+	// +optional
 	Status StorageCohortStatus `json:"status,omitempty"`
 }
 
 // StorageCohortSpec describes the attributes that a cohort is created with.
 type StorageCohortSpec struct {
 	// NodeSelector is used to specify the cohort to be considered
+	// +optional
```

Comment:
How is this optional?

Reply:
yes same question is there any use case to make it optional?